test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
Tests get run against habitrpg_test, which is [recreated when the tests get started](https://github.com/HabitRPG/habitrpg/blob/develop/test/run_tests.sh#L12). 

I've never been able to actually explore the data saved to the test db though. Users never actually appear in the collection.